tests/integration/application/application.go

```diff
@@ -32,9 +32,11 @@ var _ = Describe("application", Ordered, func() {
 	t := GinkgoT()
 
 	BeforeAll(func() {
+		var err error
 		homeDir = t.TempDir()
 		corectl = testconfig.NewCorectlClient(homeDir)
-		cfg, cfgDetails = testsetup.InitCorectl(corectl)
+		cfg, cfgDetails, err = testsetup.InitCorectl(corectl)
+		Expect(err).ToNot(HaveOccurred())
```

Comment:
There were only gomega in the beginning :)
From my PoV it was easier to write integration tests with gomega, since you can explicitly structure/mark it

Reply:
I'll stick to gomega for now, we can create a story to tackle potential re-write